readme.md

```diff
@@ -105,6 +105,8 @@ Shortcut for `--arch=all --platform=all`
 
 `icon` - *String*
 
+For OS X, this must be `.icns` format.
+
```

Comment:
(And that you don't need to set this if you're just building for Linux, see #90.)